api/pull.go

```diff
@@ -589,6 +589,14 @@ func pullHostStatus(host Node, respChan chan<- *httpResponse, dt *Dt, wg *sync.W
 	// use fake interface implementation for tests
 	timeout := time.Duration(dt.Cfg.FlagPullTimeoutSec) * time.Second
 	body, statusCode, err := dt.DtDCOSTools.Get(url, timeout)
+	if statusCode != http.StatusOK {
+		logrus.Errorf("Bad response code %d. URL %s", statusCode, url)
+		response.Status = statusCode
+		host.Health = 3
```

Comment:
is this a constant defined somewhere? should it be?

Reply:
it's not defined, there will be another PR to address it